config/mainnet.go

```diff
@@ -0,0 +1,105 @@
+package config
+
+import (
+	"math/big"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/spacemeshos/go-spacemesh/activation"
+	"github.com/spacemeshos/go-spacemesh/api/grpcserver"
+	"github.com/spacemeshos/go-spacemesh/beacon"
+	"github.com/spacemeshos/go-spacemesh/bootstrap"
+	"github.com/spacemeshos/go-spacemesh/checkpoint"
+	"github.com/spacemeshos/go-spacemesh/fetch"
+	hareConfig "github.com/spacemeshos/go-spacemesh/hare/config"
+	eligConfig "github.com/spacemeshos/go-spacemesh/hare/eligibility/config"
+	"github.com/spacemeshos/go-spacemesh/p2p"
+	"github.com/spacemeshos/go-spacemesh/syncer"
+	timeConfig "github.com/spacemeshos/go-spacemesh/timesync/config"
+	"github.com/spacemeshos/go-spacemesh/tortoise"
+)
+
+func MainnetConfig() Config {
+	return Config{
+		BaseConfig: BaseConfig{
+			DataDirParent:       defaultDataDir,
+			FileLock:            filepath.Join(os.TempDir(), "spacemesh.lock"),
+			MetricsPort:         1010,
+			MetricsPushPeriod:   60,
+			DatabaseConnections: 16,
+			NetworkHRP:          "sm",
+
+			LayerDuration:  5 * time.Minute,
+			LayersPerEpoch: 4032,
+
+			TxsPerProposal: 700,       // TODO
+			BlockGasLimit:  100107000, // 3000 of spends
+
+			OptFilterThreshold: 90,        // TODO
+			TickSize:           1_000_000, // TODO
+		},
+		Genesis: &GenesisConfig{
+			GenesisTime: "2023-07-14T08:00:00Z",
+		},
+		Tortoise: tortoise.Config{
+			Hdist:                    200,
+			Zdist:                    2,
+			WindowSize:               10000,
+			MaxExceptions:            1000,
+			BadBeaconVoteDelayLayers: 4032,
+			MinimalActiveSetWeight:   100 * 1_000_000, // TODO known units per tick size
+		},
+		HARE: hareConfig.Config{
+			N:               200,
+			ExpectedLeaders: 5,
+			RoundDuration:   25 * time.Second,
+			WakeupDelta:     25 * time.Second,
+			LimitConcurrent: 2,
+			LimitIterations: 4,
+		},
+		HareEligibility: eligConfig.Config{
+			ConfidenceParam: 200,
+		},
+		Beacon: beacon.Config{
+			Kappa:                    40,
+			Q:                        big.NewRat(1, 3),
+			GracePeriodDuration:      10 * time.Minute,
+			ProposalDuration:         4 * time.Minute,
+			FirstVotingRoundDuration: 30 * time.Minute,
+			RoundsNumber:             300,
+			VotingRoundDuration:      4 * time.Minute,
+			WeakCoinRoundDuration:    4 * time.Minute,
+		},
+		POET: activation.PoetConfig{
+			PhaseShift:        240 * time.Hour,
+			CycleGap:          12 * time.Hour,
+			GracePeriod:       1 * time.Hour,
+			RequestRetryDelay: 10 * time.Second,
+			MaxRequestRetries: 10,
+		},
+		POST: activation.PostConfig{
+			MinNumUnits:   4,
+			MaxNumUnits:   1000,
+			LabelsPerUnit: 4294967296, // TODO 64GB / 16 ?
+			K1:            26,
+			K2:            37,
+			K3:            37,
```

Comment:
I think it's not needed anymore @poszu did we deleted it for good?

Reply:
i believe it is still used, i used it while testing to make post less cpu heavyy

Reply:
As @dshulyak said, this is the number of verified labels (the proof contains K2 indices, but the verifier checks K3 of them). 

Currently, K3 should equal K2.